pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLAlbum.java

```diff
@@ -158,7 +197,40 @@ public boolean uploadImage(String title, String email, String username, String p
             e.printStackTrace();
         }
 
-        return pxlClient.makePostCall("media", body);
+        try {
+            pxlClient
+                    .getBasicrepo()
+                    .postMedia(
+                            PXLClient.apiKey,
+                            this.id,
+                            title,
+                            email,
+                            username,
+                            photoURI,
+                            approved
```

Comment:
this information was all added to the `body` object above, can't we just pass that in here instead? If not, we should get rid of the body object.

Reply:
Yes, I'll make a change.